### ISSUE TITLE
over fitting after two epoch

### ISSUE THREAD

**Issue by wxy656** [dat-ai/yolov2#6]
my training datasets contains 12300*8 data ,and my calsss is 3, my learning rate is 0.0001.  I train it with yolo-coco.weights as  initial weights,and just  after  two epoch ,the val_loss comes to impove while loss still decresce.,I think it comes out  overfitting .
Did  you meet this problem,and maybe add dropout  into net  would work?

Below is my training details

12299/12300 [============================>.] - ETA: 0s - loss: 1.6080Epoch 00001: val_loss improved from inf to 1.55635, saving model to ./backup/best_yolov2-01-1.56.weights
12300/12300 [==============================] - 2773s 225ms/step - loss: 1.6080 - val_loss: 1.5564
12299/12300 [============================>.] - ETA: 0s - loss: 1.5306Epoch 00002: val_loss improved from 1.55635 to 1.52850, saving model to ./backup/best_yolov2-02-1.53.weights
12300/12300 [==============================] - 2738s 223ms/step - loss: 1.5307 - val_loss: 1.5285
12299/12300 [============================>.] - ETA: 0s - loss: 1.4806Epoch 00003: val_loss did not improve
12300/12300 [==============================] - 2741s 223ms/step - loss: 1.4806 - val_loss: 1.5510
12299/12300 [============================>.] - ETA: 0s - loss: 1.4509Epoch 00004: val_loss did not improve
12300/12300 [==============================] - 2741s 223ms/step - loss: 1.4509 - val_loss: 1.5861
